src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/NodesDnValidator.java

```diff
@@ -0,0 +1,16 @@
+package com.amazon.opendistroforelasticsearch.security.dlic.rest.validation;
+
+import org.elasticsearch.common.bytes.BytesReference;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.rest.RestRequest;
+
+public class NodesDnValidator extends AbstractConfigurationValidator {
+
+    public NodesDnValidator(final RestRequest request, final BytesReference ref, final Settings esSettings, Object... param) {
+        super(request, ref, esSettings, param);
+        this.payloadMandatory = true;
+
+        allowedKeys.put("nodes_dn", DataType.ARRAY);
+        mandatoryKeys.add("nodes_dn");
+    }
```

Comment:
Do we need some validation here of the input as we accepting directly List<String> ?

Reply:
The nodes_dn accepts regex and wildcard similar to how the YML config is consumed. Hence chose not to validate.

Reply:
I m just worried about how bad requests are handled. Will passing any random string be accepted like `Random= testing`?